docs/elements.md

```diff
@@ -55,6 +55,7 @@ images, files, and code display. The following **decorative** elements are avail
 - [`pl-dataframe`](#pl-dataframe-element): Display DataFrames with various options.
 - [`pl-drawing`](#pl-drawing-element): Creates an image from pre-defined
   collection of graphic objects
+- [`pl-sketch`](#pl-sketch-element): Freely draw anything using a sketchpad.
```

Comment:
There was a PR a while ago that alphabetized the elements. Should move the sketch docs to maintain that ordering.